hooks/03-writable-paths.chroot

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+set -e
+# XXX: we want to migrate away from writable paths but having _some_ support of
+# this for now lets us walk before we run.
+mkdir -p /etc/system-image
+cat > /etc/system-image/writable-paths << __WRITABLE__
+# --------------------------------------------------------------------
+# 1st column: Mount point
+# 2nd column: Path relative to root of persistent storage (or auto)
+# 3rd column: type => persistent|synced|temporary
+# 4th column: action => none|transition (transition requires persistent)
+# 5th column: mount flags
+#
+# See writable-paths(5) for full details.
+# --------------------------------------------------------------------
+/etc/apparmor.d/cache                   auto                    persistent  none        none
```

Comment:
/o\ - do we need all of those?

Reply:
No, but I gave up with trimming the list. I suspect we get better result by speeding up `make update-image` and then fix this list.